### DISCUSSION TITLE
WebGL builds for v1.18.0 and v.1.19.0 not found

### DISCUSSION THREAD

**Discussion by cx20** [google/filament#5233] [Q&A] [answered]
It looks like the recently released assets doesn't include WebGL builds.

https://github.com/google/filament/releases/tag/v1.19.0

> filament-v1.19.0-web.tgz

https://github.com/google/filament/releases/tag/v1.18.0

> filament-v1.18.0-web.tgz

Can you tell us where to get the latest version of the WebGL build version?

**Comment by bejado** [comment, marked as answer]
We've had some web-related issues that have required us to skip releasing WebGL builds.

**Reply by cx20** [reply]
Thank you for teaching me. I look forward to using it again.
